src/expect_interface.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::tester::Tester;
+
+// As of now, the following expectations do not require "fn returning()" implementations and hence
+// no structure is provided for them. Setting of these expectations are built directly into tester.rs:
+
+/* proxy_log(), proxy_set_tick_period_millis(), proxy_replace_header_map_value(), proxy_remove_header_map_value()
+   proxy_add_header_map_value(), proxy_send_loca_response(), etc.
+*/
```

Comment:
Is this part of the same comment? Is this still accurate?

Reply:
Yes. I'm basically keeping track of which expectations do not require a structure in expect_interface.rs, as they don't require an associate returning function. Hence, these expectations are set directly through tester.rs Tester methods.

Reply:
I thought it might be useful to list those host-functions in case a developer notices that their corresponding expect_interface.rs structure is not present. Also, in case of future ABI changes, where host-function signatures/function might change, and thus, a structure may need to be added for one of the commented functions.